src/test/java/com/desiato/puresynth/controllers/UserControllerTest.java

```diff
@@ -57,4 +55,39 @@ public void updateUser_ShouldUpdateUserAndReturnUpdatedUser() throws Exception {
                 .andExpect(jsonPath("$.email").value(newEmail))
                 .andReturn();
     }
+
+    @Test
+    public void createUser_ShouldReturnCreatedUser() throws Exception {
+        String uniqueEmail = testAuthenticationHelper.generateUniqueEmail();
+        String newUserJson = String.format("""
+    {
+        "email": "%s",
+        "password": "password123"
+    }
+    """, uniqueEmail);
+
+        mockMvc.perform(post("/api/user")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(newUserJson))
+                .andExpect(status().isCreated())
+                .andExpect(jsonPath("$.email").value(uniqueEmail))
+                .andDo(print());
```

Comment:
?

Reply:
Are you referring to the .andDo(print()); ?
it does not provide much value, correct?

Reply:
Yes, it should not be here, after you finished debugging.